app/views/groups/group_links/_invite_group_modal.html.erb

```diff
@@ -15,16 +15,14 @@
       <div class="grid gap-4">
         <div class="form-field">
           <%= form.label I18n.t("groups.group_links.new.label.group_id") %>
-          <%= form.select :group_id,
-                      options_for_select(
-                        @namespace_linkable_groups.collect { |u| [u.name, u.id] },
-                        new_group_link.group_id
-                      ),
-                      {
-                        prompt: I18n.t("groups.group_links.new.placeholder.select_group")
-                      },
-                      required: true,
-                      "aria-label": t(:"groups.group_links.index.aria_labels.group_id") %>
+          <%= viral_select2(form:, name: :group_id) do |select| %>
+            <% @namespace_linkable_groups.each do |group| %>
+              <%= select.with_option(primary: group.name, secondary: group.id, value: group.id) %>
```

Comment:
question: Would it make more sense to display the PUIDs rather than our uuids? I just assumed users generally referenced our PUIDs when wanting to use ID as an identifier

Reply:
Yeah we want to show PUID not the ID, which is a UUID that the users never see in the application.

Reply:
Fixed in 76f7580